charts/utils/templates/portforward.yaml

```diff
@@ -0,0 +1,54 @@
+{{ if .Values.portforwards.entities }}
+{{- range $portforward := .Values.portforwards.entities }}
+{{ $prefix := printf "%s-%s" $.Values.client $.Values.environment }}
+---
+apiVersion: v1
```

Comment:
same comments apply to this one I had for the other deployment - i.e reusable snippets etc. Don't make a custom values block for portforwards use keys utils.mssql.portforward: true to evalue if this pf deployment is required or not.